.github/workflows/build.yml

```diff
@@ -30,4 +39,46 @@ jobs:
       - name: Build Leo
         id: build-leo
         run: |
-          npm run transform-tokens
\ No newline at end of file
+          npm run transform-tokens
+
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+          role-session-name: github-actions-pr-leo-${{ github.triggering_actor }}-${{ github.run_id }}
+          aws-region: us-west-2
+
+      - name: Deploy
+        id: build-site
+        env:
+          NODE_OPTIONS: "--max_old_space_size=6144"
+          AWS_REGION: us-west-2
+        run: |
+          set -eux
+          HEAD_SHA=${{ github.event.pull_request.head.sha }}
+          [ -z "$AWS_ACCESS_KEY_ID" ] && exit 1
+          BUCKET_PATH=$(jq -r .pull_request.number "$GITHUB_EVENT_PATH" | grep -E '^[0-9]+$' || :)
```

Comment:
I think `|| true` is more readable than `|| :`, especially to those less familiar with shell scripts

Reply:
but `:)` is way more fun 😄 
Alright done